velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -706,6 +706,16 @@ void PageReader::makeDecoder() {
               "DELTA_BINARY_PACKED decoder only supports INT32 and INT64");
       }
       break;
+    case Encoding::RLE:
+      switch (parquetType) {
+        case thrift::Type::BOOLEAN:
+          rleBooleanDecoder_ = std::make_unique<RleBpDataDecoder>(
```

Comment:
@jkhaliqi You can add the checks from `RleBooleanDecoder.h` in the first commit.